### DISCUSSION TITLE
Step 2 question

### DISCUSSION THREAD

**Discussion by bnlove** [TheLastGimbus/GooglePhotosTakeoutHelper#342] [I NEED HELP] [answered]
**2. Unzip them all and merge into one, so that all "Takeout" folders become one**

Within each of these Takeout folders is another folder called "Google Photos". So should I end up with one Takeout folder that contains multiple Google Photos folders? If so, should these be "Google Photos 1", "Google Photos 2", etc?

Thanks.

**Comment by TheLastGimbus** [comment, marked as answer]
everything answered in #343

also, the visual guide i drawn explains everythin
